src/github/deployment.ts

```diff
@@ -80,3 +98,36 @@ export const processDeployment = async (
 		gitHubAppId
 	);
 };
+
+const persistentSuccessDeploymentStatusToDynamoDB = async (
```

Comment:
curious why is it here and not in `deployment-service` ?

Reply:
actually I guess that makes sense to have mapping here...

Reply:
🤷